src/elements/List/List.js

```diff
@@ -62,7 +64,15 @@ function List(props) {
   const rest = getUnhandledProps(List, props)
   const ElementType = getElementType(List, props)
 
-  return <ElementType {...rest} className={classes}>{children}</ElementType>
+  if (children) {
+    return <ElementType {...rest} className={classes}>{children}</ElementType>
+  }
+
+  return (
+    <ElementType {...rest} className={classes}>
+      {_.map(items, (item) => ListItem.create(item))}
```

Comment:
Missing coverage here.  Codecov is still not updating PR status for some reason.  Killing me...